app/components/ui/label.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react"
+import * as LabelPrimitive from "@radix-ui/react-label"
+import { cva, type VariantProps } from "class-variance-authority"
+
+import { cn } from "~/utils"
+
+const labelVariants = cva(
```

Comment:
Is this just putting those classes in the className in line 18? If so, we can just inline the classes there and get rid of class-variance-authority.

Reply:
no idea this is shadcn generated code